### ISSUE TITLE
`expr.any()` is compiled incorrectly for postgres backend

### ISSUE THREAD

**Issue by ssanderson** [blaze/blaze#1510]
I have a blaze expression representing company ceo changes with the following dshape:

```
dshape("""var * {
  event_id: float64,
  trade_date: ?datetime,
  symbol: string,
  event_type: ?string,
  event_headline: ?string,
  change_status: ?string,
  change_scenario: ?string,
  change_type: ?string,
  change_source: ?string,
  change_reason: ?string,
  in_ceoname: ?string,
  in_ceogender: ?string,
  out_ceoname: ?string,
  out_ceogender: ?string,
  effective_date: ?datetime,
  event_rating: float64,
  sid: int64,
  asof_date: datetime,
  timestamp: datetime
  }""")
```

Computing the following query against the postgresql backend crashes with an error from psycopg2 indicating that the query generated invalid SQL:

```
>>> (cc.symbol == "AAPL").any()
<big stack trace>
/src/blaze/blaze/server/client.py in compute_down(expr, ec, profiler_output, **kwargs)
    185 
    186     if not ok(r):
--> 187         raise ValueError("Bad response: %s" % reason(r))
    188     response = serial.loads(content(r))
    189     if profiler_output is not None:

ValueError: Bad response: Computation failed with message:
ProgrammingError: (psycopg2.ProgrammingError) syntax error at or near "any"
LINE 1: SELECT any(eventvestor.ceo_change.symbol = 'AAPL') AS symbol...
               ^
 [SQL: 'SELECT any(eventvestor.ceo_change.symbol = %(symbol_1)s) AS symbol_any \nFROM eventvestor.ceo_change'] [parameters: {'symbol_1': 'AAPL'}]
```

**Comment by ssanderson** [comment]
cc @llllllllll

**Comment by llllllllll** [comment]
fixed with: #1511
